### DISCUSSION TITLE
First release! 🎉

### DISCUSSION THREAD

**Discussion by klaasnicolaas** [klaasnicolaas/python-powerfox#9] [Announcements]
## First release

Say hello to the first release of the Powerfox python package!
This project makes it possible to fetch data from Poweropti devices.

<hr /><em>This discussion was created from the release <a href='https://github.com/klaasnicolaas/python-powerfox/releases/tag/v0.1.0'>First release! 🎉</a>.</em>

**Comment by dailylhama** [comment]
Hi Klaas! I dicovered in the Home Assistant forum that you are interested in creating an actual integration for PowerFox and need test data. How can I get in touch with you? I could help you out there as I would be very interested in such an integration :-)

**Reply by klaasnicolaas** [reply]
In the meantime I have already had a lot of test data, but I don't have it from the heat and gas meter yet. The integration is also already in the making.

./Klaas

**Comment by dailylhama** [comment]
Okay, this is something that I can't help you with - I would only be able to provide data from the electricity meter.
Thanks for making such an integration!
